content/3_Silver/Greedy_Sorting.problems.json

```diff
@@ -76,42 +76,6 @@
       "solutionMetadata": {
         "kind": "internal"
       }
-    },
-    {
-      "uniqueId": "cses-1073",
-      "name": "Towers",
-      "url": "https://cses.fi/problemset/task/1073",
-      "source": "CSES",
-      "difficulty": "Easy",
-      "isStarred": true,
-      "tags": ["Sorted Set", "Greedy", "LIS", "Binary Search"],
-      "solutionMetadata": {
-        "kind": "internal"
-      }
-    },
-    {
-      "uniqueId": "cses-1632",
-      "name": "Movie Festival II",
-      "url": "https://cses.fi/problemset/task/1632",
-      "source": "CSES",
-      "difficulty": "Normal",
-      "isStarred": true,
-      "tags": ["Sorting", "Sorted Set", "Greedy"],
-      "solutionMetadata": {
-        "kind": "internal"
-      }
-    },
-    {
-      "uniqueId": "cses-1161",
-      "name": "Stick Division",
-      "url": "https://cses.fi/problemset/task/1161",
-      "source": "CSES",
-      "difficulty": "Hard",
-      "isStarred": false,
-      "tags": ["Greedy", "Sorted Set"],
```

Comment:
what's w/ these problems

Reply:
they got moved to sorted set or pq modules